server/src/localization.cpp

```diff
@@ -3,17 +3,94 @@
 //
 
 #include "localization.hpp"
+#include "data_store.hpp"
+
+extern insNode_t * insNoderoot;
 
 namespace ins_service
 {
 
+wifiParams_t * Localization::FillNodeDataPoints(wifiParams_t *  wifiNodeBlock, AccessPointRssiListPair mac_rssi_)
+{
+	initKalmanParams(wifiNodeBlock);
+
+	wifiNodeBlock->macAddress = (char *) mac_rssi_.first.mac_addr.c_str();
```

Comment:
C-style casts should be avoided in C++. :cry:

Reply:
Actually a cast is not needed here. Will take this out.